apps/guardian-ui/src/admin/FederationAdmin.tsx

```diff
@@ -56,60 +64,112 @@ export const FederationAdmin: React.FC = () => {
+            <Box mt='38px'>
+              <Text mb='6px' fontSize='14px' fontWeight='500' color='#344054'>
+                {t('federation-dashboard.invite-members')}
+              </Text>
+              <CopyInput
+                value={inviteCode}
+                buttonLeftIcon={<Icon as={CopyIcon} />}
+              />
+              <Text mt='6px' mb='25px' fontSize='14px' color='#6B7280'>
+                {t('federation-dashboard.invite-members-prompt')}
+              </Text>
+            </Box>
+          </Box>
+        </Flex>
         <AdminMain />
         <Flex gap={4}>
           <Card flex='1'>
-            <CardHeader>Federation info</CardHeader>
+            <CardHeader>{t('federation-dashboard.fed-info.label')}</CardHeader>
             <CardBody>
               <Table>
                 <Tbody>
                   <Tr>
-                    <Td>Your status</Td>
+                    <Td>
+                      {t('federation-dashboard.fed-info.your-status-label')}
+                    </Td>
                     <Td>{status?.server}</Td>
                   </Tr>
                   <Tr>
-                    <Td>Epoch count</Td>
+                    <Td>
+                      {t('federation-dashboard.fed-info.epoch-count-label')}
+                    </Td>
                     <Td>{epochCount}</Td>
                   </Tr>
                   <Tr>
-                    <Td>API Version</Td>
+                    <Td>
+                      {t('federation-dashboard.fed-info.api-version-label')}
+                    </Td>
                     <Td>{apiVersion}</Td>
                   </Tr>
                   <Tr>
-                    <Td>Consensus version</Td>
+                    <Td>
+                      {t(
+                        'federation-dashboard.fed-info.consensus-version-label'
+                      )}
+                    </Td>
                     <Td>{consensusVersion}</Td>
                   </Tr>
                 </Tbody>
               </Table>
             </CardBody>
           </Card>
           <Card flex='1'>
-            <CardHeader>Peer info</CardHeader>
+            <CardHeader>{t('federation-dashboard.peer-info.label')}</CardHeader>
             <CardBody>
               <Table>
                 <Thead>
                   <Tr>
-                    <Th>Name</Th>
-                    <Th>Status</Th>
+                    <Th>{t('federation-dashboard.peer-info.name-label')}</Th>
+                    <Th>{t('federation-dashboard.peer-info.status-label')}</Th>
                   </Tr>
                 </Thead>
                 {status && (
                   <Tbody>
                     {Object.entries(
-                      status?.consensus?.status_by_peer || {}
+                      status?.federation?.status_by_peer || {}
                     ).map(([peerId, peerStatus]) => (
                       <Tr key={peerId}>
-                        <Td>Peer {peerId}</Td>
+                        <Td>{peerId}</Td>
```

Comment:
```suggestion
                        <Td>Peer {peerId}</Td>
```

We should leave "Peer" until we put the actual Guardian name.
https://github.com/fedimint/ui/issues/140